src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -172,7 +198,7 @@ function verifyDeveloperNodeModules(frb: FunctionsRuntimeBundle): boolean {
   }
 
   const modBundles = [
-    { name: "firebase-admin", isDev: false, minVersion: 7 },
+    { name: "firebase-admin", isDev: false, minVersion: 8 },
```

Comment:
Do we have to drop support for `firebase-admin` 7?

Reply:
Hopefully we can support both. I'll see.

Reply:
So True or False?

Reply:
I responded to this before I decided to start being a jerk, so no True / False decision yet lol. I'll give you a boolean answer once I look into it haha.

Reply:
7 support has been restored.